### ISSUE TITLE
gmail email config looks not to work

### ISSUE THREAD

**Issue by mauog** [pyouroboros/ouroboros#318]
Hello,
I am using ouroboros in a docker container from pyouroboros/ouroboros image on a Pi
I configured an email notification using a gmail account.
Looking at the wiki gmail configuration should be already embedded in ouroboros.
Unfortunately I got this error:
```
ouroboros     | 2019-06-06 13:35:53 : DEBUG : NotifyEmail : Securing connection with STARTTLS...
ouroboros     | 2019-06-06 13:35:53 : DEBUG : NotifyEmail : Applying user credentials...
ouroboros     | 2019-06-06 13:35:53 : WARNING : NotifyEmail : A Connection error occured sending Email notification to smtp.gmail.com.
ouroboros     | 2019-06-06 13:35:53 : DEBUG : NotifyEmail : Socket Exception: (535, b'5.7.8 Username and Password not accepted. Learn more at\n5.7.8  https://support.google.com/mail/?p=BadCredentials f3sm1511017wre.93 - gsmtp')

```
The message 

**

> Username and Password not accepted

** 
is pretty clear. I double checked both username and password but they look to be OK.
I don't have double factor authentication enabled in gmail.
When ouroboros starts it looks that it well recognise the configuration:

```
ouroboros     | 2019-06-06 13:35:51 : DEBUG : NotifyEmail : Scanning first_name.last_namenumber@gmail.com against Google Mail
ouroboros     | 2019-06-06 13:35:51 : INFO : NotifyEmail : Applying Google Mail Defaults
ouroboros     | 2019-06-06 13:35:51 : DEBUG : Apprise : Loaded URL: ********iltos://first_name.last_namenumber:password@gmail.com/first_name.last_namenumber%40gmail.com?format=html&overflow=upstream&from=first_name.last_namenumber%40gmail.com&mode=starttls&smtp=smtp.gmail.com&timeout=15&user=first_name.last_namenumber%40gmail.com&verify=yes
```

I cannot figure out where is the problem.

Here is the docker-compose configuration:

```
ouroboros:
    image: pyouroboros/ouroboros
    container_name: ouroboros
    network_mode: host
    restart: always
    ports:
      - "8000:8000"
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      - DOCKER_SOCKETS="unix://var/run/docker.sock tcp://IP2:2376 tcp://IP1:2376"
      - SELF_UPDATE=true
      - DRY_RUN=false
      - LOG_LEVEL=debug
      - TZ=Europe/Rome         
- NOTIFIERS="mailtos://first_name.last_namenumber:password@gmail.com?to=first_name.last_namenumber@gmail.com"
      - CLEANUP=true
      - MONITOR="ouroboros nodered"
      - SKIP_STARTUP_NOTIFICATIONS=false

```
(NOTIFIERS is well indented in the config file, non error on docker-compose syntax)

Thank You

Regards

M.

**Comment by caronc** [comment]
You're doing it the complicated way :slightly_smiling_face: Gmail is built right into apprise mailto:// to make your life easy.

Just use the URL: `mailto://userid:app_passord@gmail.com`; [see here](https://github.com/caronc/apprise/wiki/Notify_email).

But to save you clicking the link the app password is only required if you're using two-step authentication.  You can generate one [here](https://security.google.com/settings/security/apppasswords).  Alternatively if you're not using that; just specify your normal gmail password here instead.

**Comment by mauog** [comment]
> Just use the URL: mailto://userid:app_passord@gmail.com;

It's exactly what I did but It didn't work.
Then I have activated the two factor authentication and I generate a new password for ouroboros. Now the mail notification works fine.
Perhaps the problem was that I haven't ever do any connection to gmail from my Pi directly. 
Anyway I managed to have to working just creating a password for the app.

Thank You anyway for your help
